docs/development/system-tests.md

```diff
@@ -76,13 +76,19 @@ of that class, and look at the tests in that module for examples.
 
 This test module contains several JUnit test suites:
 - QuickSystemTestSuite (the default)
-- NightlyFunctionalSystemTestSuite
-- NightlyPerformanceSystemTestSuite
+- SlowSystemTestSuite1-n
+- ExpensiveSystemTestSuite1-n
 
-Tests that are tagged as Slow or Expensive will not be included in the quick suite. The quick suite is intended to run
+Tests that are tagged as Slow(1-n) or Expensive(1-n) will not be included in the quick suite. The quick suite is intended to run
 in around 40 minutes. The nightly functional suite includes tests tagged as Slow, and will take a bit longer. The
 nightly performance suite includes all tests, including ones tagged as Expensive. The performance tests work with a
 larger bulk of data. They take time to run and can be costly to run frequently.
```

Comment:
This now refers to test suites that aren't listed. Previously this was an explanation of what those test suites are for, but it looks like you want to remove them? We could link to the other readme file to explain how to run the nightly test suites instead?

Reply:
I still think it's important to explain here briefly how these suites are run, and link to the nightly test documentation. The "1-n" parts are quite confusing as is because there's no explanation of why these are split up.

Similarly, there's no explanation of what's shown in the table for slow and expensive tests.